docs/src/main/asciidoc/spring-data-support.adoc

```diff
@@ -91,7 +113,162 @@ spring:
         database: ${AZURE_COSMOS_DATABASE}
 ----
 
-=== Samples
 
-Please refer to link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0[azure-spring-boot-samples] for more details.
+(Optional) Property `spring.cloud.azure.cosmos.consistency-level` is also supported.
+----
+spring:
+  cloud:
+    azure:
+      cosmos:
+        consistency-level: ${CONSISTENCY_LEVEL}
+----
+
```

Comment:
Actually I think this should be listed in the `Configuration` section. And I also noticed that the `Configuration` section is not listing the important configuration options we are offering, now it only shows the common ones?